test/controllers/admin/deploy_groups_controller_test.rb

```diff
@@ -198,6 +198,14 @@
 
         refute_empty deploy_group.stages.where(project: template_stage.project)
       end
+
+      it 'adds the new stage to the end of the deploy pipeline' do
+        post :create_all_stages, params: {id: deploy_group}
+
+        # the new stage is at the end of the pipeline
+        stage = deploy_group.stages.last
+        assert_equal template_stage.next_stage_ids.last, stage.id
```

Comment:
assert_equal ordering is `expected, actual` ... so this is not correct ... prefer `.must_equal` :)